src/components/utils/include/utils/qdb_wrapper/sql/sql_database.h

```diff
@@ -42,12 +42,13 @@ namespace utils {
 namespace dbms {
 
 class SQLQuery;
-
+enum class StorageType {IN_MEMORY};
 /**
  * Represents a connection to a database.
  */
 class SQLDatabase {
  public:
+  explicit SQLDatabase(StorageType);
```

Comment:
@BSolonenko Add parameter name

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/pull/17/commits/5ea679815815e48385c34414d7eac980ecd337f3